src/util/Util.js

```diff
@@ -254,15 +254,22 @@ class Util extends null {
     return text.replace(/\|\|/g, '\\|\\|');
   }
 
+  /**
+   * @typedef {Object} FetchRecommendedShardsOptions
+   * @property {number} [guildsPerShard=1000] Number of guilds assigned per shard
+   * @property {boolean} [largeBotSharding=false] Whether or not to round for large bot sharding
+   */
+
   /**
    * Gets the recommended shard count from Discord.
    * @param {string} token Discord auth token
-   * @param {number} [guildsPerShard=1000] Number of guilds per shard
+   * @param {FetchRecommendedShardsOptions} [options] Options for fetching the recommended shard count
    * @returns {Promise<number>} The recommended number of shards
    */
-  static fetchRecommendedShards(token, guildsPerShard = 1000) {
+  static fetchRecommendedShards(token, { guildsPerShard = 1000, largeBotSharding = false } = {}) {
     if (!token) throw new DiscordError('TOKEN_MISSING');
     const defaults = Options.createDefault();
+    const multiple = largeBotSharding ? 16 : 1;
```

Comment:
Personally this shouldn't take it as an option, but rather guesstimate it from what Discord says the recommended shards are (if > 150, then we are in large territory)

Reply:
Does discord tell us on the API side what the requirement is for large bot sharding? If not I'm not sure guesstimating here would be good, large bot sharding requirements were originally 250k, then 200k, and now 150k iirc. I feel like giving the user an option is better than just guesstimating since the requirement could change in the future.

Reply:
That's also a double edged sword, especially if you use the sharding manager which doesn't (iirc) let you pass in this option. Personally I'd rather have to decrement a >= check but guarantee users that they'll get the shards they should have! @discordjs/the-big-3 whats your take on thios?

Reply:
We can default to 16 but should allow passing any arbitrary number in case this requirement ever changes.

Reply:
> We can default to 16 but should allow passing any arbitrary number in case this requirement ever changes.

Okay, so assuming the FetchRecommendedShardsOption stays we can either simply add a largeBotShardMultiple like:
```
guildsPerShard = 1000
largeBotSharding = false
largeBotShardingMultiple = 16
```

Or just ditch the largeBotSharding option entirely by replacing it with a multiple
```
guildPerShard = 1000
roundToMultiple = 1 (and state in the documentation it needs to be set to 16 for large bot sharding)
```

Reply:
I'd rather go with the 2nd option 👍

Reply:
I'd name it `multipleOf`, since it's not rounding, but ceiling to the next multiple.

Reply:
Sounds good, I'll make these changes later today.